core/backend/src/IModelDb.ts

```diff
@@ -1178,6 +1179,36 @@ export abstract class IModelDb extends IModel {
   public async generateElementGraphics(request: ElementGraphicsRequestProps): Promise<Uint8Array | undefined> {
     return generateElementGraphics(request, this);
   }
+
+  private static _settingPropNamespace = "settings";
+  public saveSettingDictionary(name: string, dict: SettingDictionary) {
```

Comment:
@calebmshafer @wgoehrig This is how i suggest we save Settings that specify where to find reality data associated with an iModel (for example).